src/CommandingOfficers/Commander.java

```diff
@@ -357,17 +357,17 @@ public GameEventQueue receiveBattleEvent(final BattleSummary summary)
     double myHPDealt = -10;
     if( this == summary.attacker.CO )
     {
-      minion = summary.attacker;
-      myHPLoss = summary.attackerHPLoss;
-      enemy = summary.defender;
-      myHPDealt = summary.defenderHPLoss;
+      minion = summary.attacker.unit;
+      myHPLoss = summary.attacker.deltaHP * -1;
+      enemy = summary.defender.unit;
+      myHPDealt = summary.defender.deltaHP * -1;
```

Comment:
Oh. This `*-1` business is everywhere. Maybe have a way to ask for HP lost? Magic numbers are messy.